extendeddatacrossword.go

```diff
@@ -138,8 +138,9 @@ func (eds *ExtendedDataSquare) solveCrosswordRow(
 
 	// Check that rebuilt shares matches appropriate root
 	err = eds.verifyAgainstRowRoots(rowRoots, uint(r), rebuiltShares)
+	var byzErr *ErrByzantineData
 	if err != nil {
-		if byzErr, ok := err.(*ErrByzantineData); ok {
+		if errors.As(err, &byzErr) {
```

Comment:
Move inside if case?

Reply:
wdym?

Reply:
I mean byzErr should be declared after error check and not before

Reply:
Done